### DISCUSSION TITLE
看中美髮店可以「釘」住客人！螢幕導入AI偵測性別、年齡，如何精準投廣告？

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#456] [General]
看中美髮店可以「釘」住客人！螢幕導入AI偵測性別、年齡，如何精準投廣告？
如果能精準地投放廣告，真的會讓廣告產生效果的效率高很多，Google就做得很好哈哈哈
https://www.technice.com.tw/ai/1104/

**Comment by brian431** [comment]
> 看中美髮店可以「釘」住客人！螢幕導入AI偵測性別、年齡，如何精準投廣告？
> 
> 如果能精準地投放廣告，真的會讓廣告產生效果的效率高很多，Google就做得很好哈哈哈
> 
> https://www.technice.com.tw/ai/1104/

最近看了一部很不錯的電影《關鍵報告》，裡面其中一個場景便是非常個人化的廣告。
看來我們是離這種未來越來越接近了。

https://youtu.be/7bXJ_obaiYQ

**Comment by Chenszuhong** [comment]
用AI去客製化廣告確實是一個滿不錯的想法

**Comment by limitx0** [comment]
如果是有興趣的廣告還好
不然對我來說去個美髮店還要看廣告很討厭

**Comment by Ben6uo** [comment]
有點好奇會不會出現一些18+的廣告
